src/stactools/cop_dem/stac.py

```diff
@@ -42,18 +45,37 @@ def create_item(href: str,
     if m:
         if m.group(1) == '30':
             gsd = 90
+            collection = f"cop-dem-glo-{gsd}"
         elif m.group(1) == '10':
             gsd = 30
+            collection = f"cop-dem-glo-{gsd}"
         else:
             raise ValueError("unknown resolution {}".format(m.group(1)))
         title = m.group(2)
     else:
         raise ValueError("unable to parse {}".format(href))
 
-    item.add_links(COP_DEM_LINKS)
-    item.common_metadata.platform = COP_DEM_PLATFORM
+    item.add_links(co.COP_DEM_LINKS)
+    # Valid stac item requires collection
+    # collection requires a matching Link object
+    item.collection_id = collection
+    root_href = ''  # TODO: should this become and optional arg?
+    item.links.append(
+        Link(rel="collection",
+             target=os.path.join(root_href, f"{collection}.json")))
```

Comment:
```suggestion
```

I'd suggest removing this section. Items are usually wired into collections by a higher-level script after item creation -- different platforms might end up having different values for the collection id field, for example.

Reply:
removed